json.go

```diff
@@ -55,6 +55,18 @@ func (p Prefix) toJSON() ([]byte, error) {
 	return pj, nil
 }
 
+func (ps Prefixes) toJSON() ([]byte, error) {
+	var pfxjs []prefixJSON
```

Comment:
Why not just
```
	pj, err := json.Marshal(ps)
	if err != nil {
		return nil, fmt.Errorf("unable to marshal prefixes:%w", err)
	}
	return pj, nil
```
?

Reply:
Because availableChildPrefixes and ips in Prefix are not exported and the json.(Un)marshaller has no direct access . With uppercase (exported) names, it would work ...
So I guess this copying to/from jsonPrefix struct is indeed needed

Reply:
As this is not a time critical API endpoint, i would keep it as it is.